system-addon/common/PrerenderData.jsm

```diff
@@ -0,0 +1,42 @@
+const prefConfig = {
+  // Prefs listed with "invalidates: true" will prevent the prerendered version
+  // of AS from being used if their value is something other than what is listed
+  // here.
```

Comment:
As we were discussing yesterday, some of these have minor consequences, which we currently believe are acceptable.  Documenting those consequences and tradeoffs here would help maintainability.